flink-core/pom.xml

```diff
@@ -80,6 +80,13 @@ under the License.
 			<!-- managed version -->
 		</dependency>
 
+		<!-- For multiset -->
+		<dependency>
+			<groupId>org.apache.commons</groupId>
```

Comment:
We should not add additional dependencies to Flink just because of a new data type. There is also no reason behind choosing this library. Couldn't we not just use a usual Java Map? Otherwise I would propose to add class for our own type like we did it for `org.apache.flink.types.Row`. Calcite is using `List`, which is not very nice, but would also work.

Reply:
Thanks. Use java.util.Map instead.